pkg/cmd/roachtest/tests/kv.go

```diff
@@ -488,13 +488,13 @@ func registerKVQuiescenceDead(r registry.Registry) {
 			})
 			// Graceful shut down third node.
 			m.ExpectDeath()
-			gracefulOpts := option.DefaultStopOpts()
-			gracefulOpts.RoachprodOpts.Sig = 15 // SIGTERM
-			gracefulOpts.RoachprodOpts.Wait = true
-			gracefulOpts.RoachprodOpts.MaxWait = 30
-			c.Stop(ctx, t.L(), gracefulOpts, c.Node(len(c.CRDBNodes())))
-			// If graceful shutdown fails within 30 seconds, proceed with hard shutdown.
-			c.Stop(ctx, t.L(), option.DefaultStopOpts(), c.Node(len(c.CRDBNodes())))
+			if err := c.StopE(
+				ctx, t.L(), option.NewStopOpts(option.Graceful(shutdownMaxWait)), c.Node(len(c.CRDBNodes())),
```

Comment:
Keep the timeout as 30 seconds instead of 300?

Reply:
Good catch, fixed.